server/src/test/java/org/apache/druid/server/log/FilteredRequestLoggerTest.java

```diff
@@ -106,6 +113,10 @@ public void testNotFilterAboveThreshold() throws IOException
     EasyMock.expect(nativeRequestLogLine.getQueryStats())
             .andReturn(new QueryStats(ImmutableMap.of("query/time", 1000)))
             .once();
+    EasyMock.expect(nativeRequestLogLine.getQuery())
+            .andReturn(new SegmentMetadataQuery(new LegacyDataSource("foo"),
```

Comment:
Please extract `new SegmentMetadataQuery(...)` as a variable.

Reply:
Wasn't sure if I should do one final class variable or two local variables (one per method). Went with the former, but would be easy to change to something else if my choice happens to be considered a bad pattern in Java